### ISSUE TITLE
disable that the food diminishes

### ISSUE THREAD

**Issue by malleux** [squeek502/SpiceOfLife#169]
i still want the mod, because without it my game crashes but i dont want the thing where i have to eat different food everytime, its really annoying sometimes, is there a way to disable just that?

**Comment by squeek502** [comment]
Yes, the config option is `"food.modifier.enabled"`. No clue why your game would crash without this mod though.
